### PR TITLE
Integrated feedback from EGG (Mandy, George, and Johan)

### PR DESCRIPTION
**Pull request by amycoffin**

### CODE REVIEW

File: website/content/faqs/what-is-enigma-and-how-does-it-determine-variant-classifications.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+ENIGMA (Evidence-based Network for the Interpretation of Germline Mutant Alleles) is an international consortium of researchers who provide expert opinions on the clinical significance of variation in BRCA1, BRCA2, and other breast cancer genes. This consortium serves as the BRCA expert review body of ClinGen. ENIGMA uses classification criteria tuned specifically for the BRCA genes to evaluate the clinical significance of BRCA1/2 variants. Evidence reviewed includes many different types of data, including population frequencies, clinical information, and epidemiological measures. This data also includes effects of variation on transcript and protein structure and function.  All available information is combined to estimate the probability that a variant is cancer-causing, using qualitative (rules-based) and quantitative (statistical) method multi-factorial likelihood analysis.
```
Comment (zfisch) [inline website/content/faqs/what-is-enigma-and-how-does-it-determine-variant-classifications.md]: I noticed some sentences have one space after a period and others have 2 spaces after a period. It seems like this should be more consistent.

E.G.
`review body of ClinGen. ENIGMA uses`
vs
`function.  All available `

File: website/content/help.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ To begin, notice the search bar on the home page. This search bar accesses our d
...
 When you have identified your variant of interest in the list, you can click anywhere in the variant’s data row to access the Variant Detail Page. The Variant Detail Page provides an organized summary of the searched variant’s information, including its aliases, its clinical significance, the date it was last evaluated, and other relevant data. Information is grouped into tiles for your convenience.
 
+Please note that searching for a variant using a genomic coordinate will return all variants that match according to any of the hg38, hg19, or hg18 genome builds. For example, if you search for a variant using its hg38 coordinate, and it happens to match the coordinate of some variant  in the hg19 build, both variants will be returned in the search. In this case, make sure to verify that the coordinate(s) and genome build are correct once you navigate to the Variant Details Page.
```
Comment (zfisch) [inline website/content/help.md]: Extra space in `some variant  in the hg19 build`

File: website/content/help.md [hunk 3]
```diff
@@ -28,15 +30,15 @@ The Expert Reviewed Data Portal contains an abridged view of the variant data in
...
-* _HGVS Nomenclature_: A nomenlcature system standardized by the Human Genome Variation Society \(HGVS\) which  utilizes coordinates to name variants across genomic DNA, coding DNA, RNA, and protein molecules. Each coordinate provides an indicator of the biomolecule being referred to \('c.', 'g.', 'p.'\), a location in that biomolecule \('12345'\), and an indication of what changed in the variant \('G&gt;A', 'Asp...Asn'\). Most variant aliases utilize this standard nomenclature:
-  * _HGVS Nucleotide:_ HGVS variant alias which references the nucleotide change based ont he location in the coding DNA, not the genomic DNA.
-  * _HGVS RNA_: The variant alias that uses RNA location
+* _HGVS Nomenclature_:A nomenclature system standardized by the Human Genome Variation Society \(HGVS\) which utilizes coordinates to name variants across genomic DNA, coding DNA, RNA, and protein molecules. Each coordinate provides an indicator of the biomolecule being referred to \('c.', 'g.', 'p.'\), a location in that biomolecule \('12345'\), and an indication of what changed in the variant \('G&gt;A', 'insA', 'dupT', 'del15', 'Asp...Asn'\). Most variant aliases utilize this standard nomenclature:
```
Comment (zfisch) [inline website/content/help.md]: Why was the space removed after the colon in: `_HGVS Nomenclature_:A nomenclature`

Comment (zfisch) [reply website/content/help.md]: Also, is this sentence supposed to end with a colon? `Most variant aliases utilize this standard nomenclature:`

Comment (amycoffin) [reply website/content/help.md]: Correct, it is supposed to end in a colon.

File: website/content/faqs/what-is-enigma-and-how-does-it-determine-variant-classifications.md (hunk 1 above)
Comment (amycoffin) [reply website/content/faqs/what-is-enigma-and-how-does-it-determine-variant-classifications.md]: Thanks for bringing the double spaces to my attention. I will attend to them now, and re-push ASAP.

**Comment by amycoffin** [comment]
Typos have been addressed.

**Pull request merged**